clients/docs/includes/configs/cloud/hocon.config

```diff
@@ -15,6 +15,8 @@ kafka {
     group.id = "consumer"
     key.deserializer = org.apache.kafka.common.serialization.StringDeserializer
     value.deserializer = org.apache.kafka.common.serialization.StringDeserializer
+    # Increase session timeout for higher availability (45s is the default for Apache Kafka 3.0 and above)
```

Comment:
For consistency, this should also use the new verbiage `Best practice for higher availability in ........`

Reply:
strange that got missed in the previous commit - foggy brain 😞